dist/js/social-share-kit.js

```diff
@@ -216,6 +216,9 @@ var SocialShareKit = (function () {
                 }
             };
         switch (network) {
+            case 'whatsapp':
+                url = 'https://api.whatsapp.com/send?text=' + encodeURIComponent(title + '\n' + shareUrl + '\n\n' + text + '\n');
```

Comment:
should use this rather
https://wa.me/?text=

Reply:
why ?
https://api.whatsapp.com/send?text=
works as it should be.

Reply:
If I remember correctly this wasn't working on Android or IOS

Reply:
i did a test on 
https://wa.me/?text=
its redirect to 
https://api.whatsapp.com/send?text=
on ios & android